src/components/Image/index.js

```diff
@@ -60,10 +60,10 @@ export class Image extends Component {
     return `data:image/svg+xml,%3Csvg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 ${width} ${height}"%3E%3C/svg%3E`
   }
 
-  handleLoad = async () => {
+  async handleLoad() {
```

Comment:
I think you need to bind it in the constructor then, otherwise this.setState will be undefined

Reply:
Worked fined without doing that, but changed it. Fine for now, maybe gonna refactor this component to use Hooks instead this Friday ✌️